### PR TITLE
Create 2022-02-25-build-clis-with-quo.md

### PR DESCRIPTION
**Pull request by gerrishons**
Tutorial on how to build CLIs using quo

### CODE REVIEW

File: Content/posts/2022-02-25-build-clis-with-quo.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+- Using ``quo.prompt.Prompt`` object
+
+```python
+ from quo.prompt import Prompt
+   
+ session = Prompt()
+ session.prompt("Type something:") 
+```
+Read more on [Prompt](https://quo.readthedocs.io/latest/prompt.html)
```
Comment (python-engineer) [inline Content/posts/2022-02-25-build-clis-with-quo.md:89]: this link does not work

File: Content/posts/2022-02-25-build-clis-with-quo.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+# Quo Library
+Quo contains a number of builtin features you can use to create elegant output in your CLI.
+
+Click the following headings for details:»
+<details>
```
Comment (python-engineer) [inline Content/posts/2022-02-25-build-clis-with-quo.md:95]: please remove this details view and just use normal h2 headings (##) and text.

File: Content/posts/2022-02-25-build-clis-with-quo.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+
+ data = [
+   ["Name", "Gender", "Age"],
+   ["Alice", "F", 24],
+   ["Bob", "M", 19],
+   ["Dave", "M", 24]
+ ]
+ echo(Table(data))
+```
+![tabulate](https://github.com/secretum-inc/quo/raw/master/pics/tabulate.png)
```
Comment (python-engineer) [inline Content/posts/2022-02-25-build-clis-with-quo.md:245]: this image is not showing

**Review by python-engineer — CHANGES_REQUESTED** [review]

**Review by gerrishons — COMMENTED** [review]
The changes have been updated...

**Pull request merged**